src/views/Transcript/index.js

```diff
@@ -44,15 +49,20 @@ export default class Transcript extends Component {
     }
   }
 
-  //Compares each activity from TranscriptInfo to determine how to group by session
+  //Compares activity from activityList to previous activity' session to to determine how to group
```

Comment:
Please end this line with a period, so it doesn't read like "... how to group isUnique ..."
(Next line also needs a period.)